docs/release_notes/v0.11.0.md

```diff
@@ -0,0 +1,489 @@
+
+**Updated** state store and secrets store examples
+
+### .NET SDK
+
+**Added** metadata support for State APIs
+
+**Added** support for State Transaction API 
+
+**Added** ability to specify default DaprClient options to AddDapr (Thanks @EdwinVW)
+
+**Added** support to send headers as metadata to sidecar (Thanks @amolenk)
+
+### Go SDK
+
+**Added** sample application for pub/sub (Thanks @abserari)
+
+**Fixed** README documentation (Thanks @li1234yun)
+
+**Added** support for pub/sub event handling code, retry logic
+
+**Added** consistent support for parent trace ID in client requests
+
+**Added** explicit Dapr API authentication token support
+
+**Added** metadata support for State APIs
+
+**Added** code coverage GitHub action 
+
+### Python SDK
+
+**Added** support for Bulk Get State API
+
+**Added** example for scoping of secrets 
+
+**Added** support for State Transaction API 
+
+**Added** metadata support for State APIs
+
+**Added** support to save multiple states
+
+### Rust SDK
+
+**Added** metadata support for State APIs (Thanks @gdhuper)
+
+### JavaScript SDK
+
+**Added** metadata support for State APIs
+
+### Docs
+
+**Fixed** kubectl command to apply correct yaml file (Thanks @awolf)
+
+**Fixed** broken links to binding docs (Thanks @awolf)
+
+**Added** docs for changes to local secret store 
+
+**Updated** actors API reference doc
+
+**Updated** pub/sub API reference doc
+
+**Updated** state API reference doc
+
+**Fixed** broken links (Thanks @sau-lanvy)
+
+**Added** cross namespace information for service invocation 
+
+**Added** declarative pub/sub documentation 
+
+**Updated** documentation for Prometheus and Grafana setup
+
+**Added** metrics spec information 
+
+**Added** disclaimer about plain text secrets 
+
+**Added** Redis failover guidelines (Thanks @abserari)
+
+**Added** secrets scoping information 
+
+**Fixed** missing line continuation in commands 
+
+**Added** configuration options doc for decodeBase64 (Thanks @mtaghavi2005)
+
+**Updated** Helm chart links 
+
+**Updated** NATS streaming setup and deleted NATS setup (Thanks @abserari)
+
+**Added** case-sensitivity of input parameters (state keys, pubsub topics)
+
+**Added** clarification on use of CloudEvents during publish 
+
+**Updated** Azure Blob Storage binding documentation 
+
+**Refctored** service invocation and state management cocncepts doc
+
+**Updated** KEDA HowTo doc
+
+## Upgrading to Dapr 0.11.0
+
+If you're upgrading from an older version of Dapr to 0.11.0, follow the steps here to ensure a smooth upgrade. You know, the one where you don't get red errors on the terminal.. we all hate that, right?
+
+### Local Machine / Self-hosted
+
+Uninstall Dapr using the CLI you currently have. Note that this will remove the default $HOME/.dapr directory, binaries and all containers dapr_redis, dapr_placement and dapr_zipkin. Linux users need to run sudo if they have installed binary in default path /usr/local/bin/ or have docker command needing sudo.:
```

Comment:
```suggestion
Uninstall Dapr using the CLI you currently have. Note that this will remove the default $HOME/.dapr directory, binaries and all containers dapr_redis, dapr_placement and dapr_zipkin. Linux users need to run sudo if they have installed binary in default path /usr/local/bin/ or have docker command needing sudo:
```